motr/init.c

```diff
@@ -32,6 +32,11 @@
 #include "net/net.h"
 #include "net/bulk_emulation/mem_xprt.h"
 #include "net/lnet/lnet.h"
+#ifndef __KERNEL__
```

Comment:
Do we need this #ifndef __KERNET__

Reply:
Yes. To ensure that these files are not compiled when we are building kernel modules.